src/table_builder.cc

```diff
@@ -84,9 +79,9 @@ void TitanTableBuilder::Add(const Slice& key, const Slice& value) {
     auto blob_file = storage->FindFile(index.file_number).lock();
     if (ShouldMerge(blob_file)) {
       BlobRecord record;
-      PinnableSlice buffer;
-      Status s = storage->Get(ReadOptions(), index, &record, &buffer);
-      if (s.ok()) {
+      Status get_status = GetBlobRecord(index, &record);
+
+      if (get_status.ok()) {
```

Comment:
if not okay, maybe we should set `status_`  and return, or it will neglect error silently.

Reply:
If not ok it will write original key-value_index to base_builder instead of doing level_merge

Reply:
Early return make the logic hard to read. We should refactor the code again in some time. But for now, please add a comment to describe the behavior.

Reply:
I'm afraid of neglect error silently makes us hardly know whether level-merge/fallback working properly? maybe print some logs? @yiwu-arbug

Reply:
make sense. log per key is too noisy though. how about adding an error count and print at the end?

Reply:
looking good. and please add a debug log for error message

Reply:
> looking good. and please add a debug log for error message

@wujy-cs mind address the comment? Thanks.

Reply:
@yiwu-arbug 
Ok.
I didn't record erorr for fallback since it's normal to read a GC-ed blob file and get error during fallback.